Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -4582,7 +4583,8 @@ def com_google_fonts_check_integer_ppem_if_hinted(ttFont):
                 "\n"
                 "# install gftools\n"
                 "pip install git+https://www.github.com"
```

Comment:
let's take this opportunity to decide whether to keep these instructions to install from git, or to change it to installing a stable release of gftools instead.